deployment/helm/templates/onvif.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.onvif.enabled }}
+apiVersion: akri.sh/v0
+kind: Configuration
+metadata:
+  name: {{ .Values.onvif.name }}
+spec:
+  protocol:
+    onvif:
+      ipAddresses: 
+        action: {{ .Values.onvif.ipAddresses.action }}
+        {{- if .Values.onvif.ipAddresses.items}}
+        items:
+        {{- toYaml .Values.onvif.ipAddresses.items | nindent 8 }}
+        {{- else }}
+        items: []
+        {{- end }}
+      macAddresses:
+        action: {{ .Values.onvif.macAddresses.action }}
+        {{- if .Values.onvif.macAddresses.items}}
+        items:
+        {{- toYaml .Values.onvif.macAddresses.items | nindent 8 }}
+        {{- else }}
+        items: []
+        {{- end }}
+      scopes:
+        action: {{ .Values.onvif.scopes.action }}
+        {{- if .Values.onvif.scopes.items}}
+        items:
+        {{- toYaml .Values.onvif.scopes.items | nindent 8 }}
+        {{- else }}
+        items: []
+        {{- end }}
+      discoveryTimeoutSeconds: {{ .Values.onvif.discoveryTimeoutSeconds }}
+  {{- if .Values.onvif.brokerPod.image.repository }}
+  # Only add broker pod spec if a broker image is provided
+  brokerPodSpec:
+    containers:
+    - name: {{ .Values.onvif.name }}-broker
+      image: {{ .Values.onvif.brokerPod.image.repository | quote }}
+      imagePullPolicy: {{ .Values.onvif.brokerPod.image.pullPolicy }}
+      {{- if .Values.onvif.brokerPod.env }}
+      # Optionally set environment variables in the broker pods. 
+      # Add the environment variable keys and values into an env.yaml
+      # file and apply to your Helm install command with `-f env.yaml`.
+      # Example env.yaml file:
+      # # onvif:
+      # #   brokerPod:
+      # #     env:
+      # #       ENV_KEY1: VALUE_1
+      env:
+        {{- range $key, $val := .Values.onvif.brokerPod.env }}
+        - name: {{ $key }}
+          value: {{ $val | quote }}
+        {{- end }}
+      {{- end }}
+      resources:
```

Comment:
Its fine if this is out of scope, but will ask. Should we support specification of other requires/limits? Possibly this could be relegated to folks using custom configuration, instead of the precreated template.

Reply:
We have [documentation on requesting other limits](https://github.com/deislabs/akri/blob/main/docs/modifying-akri-installation.md#modifying-the-brokerpodspec). I think if we find people creating issues for this/ asking for it to be configurable via helm values, we could add it.

Reply:
Sounds good.